modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -25,30 +25,167 @@ Before you install an {product-title} cluster on your vCenter that uses infrastr
+|`InventoryService.Tagging.DeleteTag`
+|`InventoryService.Tagging.EditCategory`
+|`InventoryService.Tagging.EditTag`
+|`Sessions.ValidateSession`
+|`StorageProfile.View`
+
+.5+|vSphere vCenter Cluster
+.5+|Always
+|`Host.Config.Storage`
+|`Resource.AssignVMToPool`
+|`VApp.AssignResourcePool`
+|`VApp.Import`
+|`VirtualMachine.Config.AddNewDisk`
+
+.3+|vSphere Datastore
+.3+|Always
+|`Datastore.AllocateSpace`
+|`Datastore.Browse`
+|`Datastore.FileManagement`
+
+|vSphere Port Group
+|Always
+|`Network.Assign`
+
+.26+|Virtual Machine Folder
+.26+|Always
+|`Resource.AssignVMToPool`
+|`VApp.Import`
+|`VirtualMachine.Config.AddExistingDisk`
+|`VirtualMachine.Config.AddNewDisk`
+|`VirtualMachine.Config.AddRemoveDevice`
+|`VirtualMachine.Config.AdvancedConfig`
+|`VirtualMachine.Config.Annotation`
+|`VirtualMachine.Config.CPUCount`
+|`VirtualMachine.Config.DiskExtend`
+|`VirtualMachine.Config.DiskLease`
+|`VirtualMachine.Config.EditDevice`
+|`VirtualMachine.Config.Memory`
+|`VirtualMachine.Config.RemoveDisk`
+|`VirtualMachine.Config.Rename`
+|`VirtualMachine.Config.ResetGuestInfo`
+|`VirtualMachine.Config.Resource`
+|`VirtualMachine.Config.Settings`
+|`VirtualMachine.Config.UpgradeVirtualHardware`
+|`VirtualMachine.Interact.GuestControl`
+|`VirtualMachine.Interact.PowerOff`
+|`VirtualMachine.Interact.PowerOn`
+|`VirtualMachine.Interact.Reset`
+|`VirtualMachine.Inventory.Create`
+|`VirtualMachine.Inventory.CreateFromExisting`
+|`VirtualMachine.Inventory.Delete`
+|`VirtualMachine.Provisioning.Clone`
+
+.26+|vSphere vCenter Datacenter
+.26+|If the installation program creates the virtual machine folder
+|`Resource.AssignVMToPool`
+|`VApp.Import`
+|`VirtualMachine.Config.AddExistingDisk`
+|`VirtualMachine.Config.AddNewDisk`
+|`VirtualMachine.Config.AddRemoveDevice`
+|`VirtualMachine.Config.AdvancedConfig`
+|`VirtualMachine.Config.Annotation`
+|`VirtualMachine.Config.CPUCount`
+|`VirtualMachine.Config.DiskExtend`
+|`VirtualMachine.Config.DiskLease`
+|`VirtualMachine.Config.EditDevice`
+|`VirtualMachine.Config.Memory`
+|`VirtualMachine.Config.RemoveDisk`
+|`VirtualMachine.Config.Rename`
+|`VirtualMachine.Config.ResetGuestInfo`
+|`VirtualMachine.Config.Resource`
+|`VirtualMachine.Config.Settings`
+|`VirtualMachine.Config.UpgradeVirtualHardware`
+|`VirtualMachine.Interact.GuestControl`
+|`VirtualMachine.Interact.PowerOff`
+|`VirtualMachine.Interact.PowerOn`
+|`VirtualMachine.Interact.Reset`
+|`VirtualMachine.Inventory.Create`
+|`VirtualMachine.Inventory.CreateFromExisting`
+|`VirtualMachine.Inventory.Delete`
+|`VirtualMachine.Provisioning.Clone`
+|`Folder.Create`
+|`Folder.Delete`
+|===
+====
+
+
+Additionally, the user requires some `ReadOnly` permissions, and some of the roles require permission to propogate the permissions to child objects. These settings vary depending on whether or not you install the cluster into an existing folder.
+
+.Required permissions and propagation settings
+[%collapsible]
+====
+[cols="3a,3a,3a,3a",options="header"]
```

Comment:
With 4 columns, I would just double check this after it's merged, to make sure that the table doesn't look awful on the customer portal :smile: